.env_template

```diff
@@ -0,0 +1 @@
+API_KEY=""
```

Comment:
Maybe add a comment to explain that this is the GitHub webhook secret? (or rename the variable)